README.md

```diff
@@ -81,27 +80,29 @@ wildfly:
      message: "Too many commits in PR!"
 ```
 
-1. `title-check`- Checks the title of a PR by using a regular expression in the `pattern` field.
+1. `title`- Checks the title of a PR by using a regular expression in the `pattern` field.
 > The correct format in example is "[WFLY-11] Name"
 2. `description`- Checks comments of a PR by using individual regular expressions in the `pattern` fields under `regexes`.
 > The correct format in example is "https://issues.jboss.org/browse/WFLY-11"
 3. `commits-quantity`- Checks the amount of commits in PR with the amount in the `quantity` field.
 > In the field you can use the exact values '1', '2' or range '1-2', '2-4' up to 100.
 4. `message` - The text of an error message in the respective check.
 
+> **_NOTE:_**  `title` and `commit-message` are enabled by default. More [here](wildfly-bot-config-example.yml).
+
 Also, there is a possibility to select checks that you need. Just left in the `wildfly-bot.yml` file checks you need.
 
 Like this:
 ```
 wildfly:
+ projectKey: "WFLY"
```

Comment:
since this is optional, I would leave from the simple example

Reply:
Sure, I wanted to display this one option as one that would be almost always changed